src/components/modules/admin/hooks/useLoanOfferRequests.hook.ts

```
@@ -0,0 +1,93 @@
+  };
+
+  const handleApprove = async (offerId: string): Promise<void> => {
+    try {
+      const res = await approveLoanOffer({ offerId });
+      if (res.success) {
+        toast.success("Offer approved successfully");
+        setLoanOffers((prev) => prev.filter((o) => o.id !== offerId));
+      } else {
+        toast.error("Failed to approve offer");
+      }
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)